packages/blade/src/components/ChatBubble/types.ts

```diff
@@ -0,0 +1,42 @@
+import type React from 'react';
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import type { DataAnalyticsAttribute, StringChildrenType, TestID } from '~utils/types';
+
+type CommonChatBubbleProps = {
+  isLoading?: boolean;
+  validationState?: 'error' | 'none';
+  errorText?: string;
+  onClick?: () => void;
+  footerActions?: React.ReactNode;
+  children?: React.ReactNode | StringChildrenType;
```

Comment:
If we are adding React.ReactNode do we need to explicitly mention StringChildrenType?

Reply:
seems to work fine. removing it